### PR TITLE
Brand new design

### PR DESCRIPTION
**Pull request by lukecarbis**

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-wpaustralia.org
```
Comment (ntwb) [inline CNAME]: We need to keep this so we can continue to host this site at wpaustralia.org

**Review by ntwb — CHANGES_REQUESTED** [review]
LGTM @lukecarbis, I've just one issue, we need to keep the CNAME domain name alias file.

**Comment by lukecarbis** [comment]
Whoops! Done.

**Comment by ntwb** [comment]
Thanks a bunch Luke 💐

**Comment by lukecarbis** [comment]
You're welcome.

**Pull request merged**